SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -132,6 +132,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, StartupLogRate(5.0f)
 	, ActorMigrationLogRate(5.0f)
 	, bEventTracingEnabled(false)
+	, SamplingProbability(1.0f)
```

Comment:
Should we choose a better default here? Tracing everything atm is not scalable at all

Reply:
I guess this may change when we integrate better worker tools?

Reply:
Maybe. Not sure on a nice default. Recommendation?

Reply:
Eh maybe 1 makes sense for now, we can always adjust it later when we have more info